community/README.adoc

```diff
@@ -0,0 +1,47 @@
+= Community contribution and involvement
+
+== JIRA usage
+
+See link:how_to_use_jira.adoc[How To Use JIRA]
+
+=== JIRA Filters
+
+To facilitate sharing of consistent JIRA queries, we now have this:
+
+https://github.com/maxandersen/jiralint/blob/filter_prototype/filters.json
```

Comment:
Changed. I wish you could have provided that detail in your first comment so as to avoid followup questions. :(